builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -377,6 +378,11 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 							Default:  false,
 							ForceNew: true,
 						},
+
+						"controller_type": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
if `scsi` is the default, then we should specify that here and now have to do the if / else on it further down. We just need to document it is `scsi`